cmsdb/processes/hhh.py

```diff
@@ -5,14 +5,15 @@
 """
 
 __all__ = [
-    "hhh", "hhh_ggf", "hhh_ggf_4b2tau",
+    "hhh", "hhh_ggf", "coupling_combinations",
```

Comment:
we should also include all the subprocesses that are added in the loop over the `coupling_combinations`